### ISSUE TITLE
nextcloud >= 21 needs apc.enable_cli = 1

### ISSUE THREAD

**Issue by jankowa** [debops/debops#1759]
occ command on nextcloud >= 21.0.3 fails with 

```
occ
An unhandled exception has been thrown:
OC\HintException: [0]: Memcache \OC\Memcache\APCu not available for local cache (Is the matching PHP module installed and enabled?)
```

The option `apc.enable_cli = 1`  has to be present. But it looks like it has been only enabled for owncloud 8.1 (https://github.com/debops/debops/commit/bfbfb9227658f8ae940b0a5cc1db893267808c64). The file `/cli/conf.d/enable_apc.ini` is removed on the task `- name: Remove legacy files` in `tasks/setup_owncloud.yml`

I know that nextcloud 21 is not supported official. But my suggestion would be to add: 

```owncloud__php__dependent_configuration:
    
  - filename: '10-owncloud'    
    by_role: 'debops.owncloud'
    state: '{{ "present" if (((owncloud__apcu_enabled|bool) and (owncloud__release is match("8\.1"))) or 
            ((owncloud__variant in ["nextcloud"]) and
            (owncloud__release is version_compare("21.0", ">="))))
            else "absent" }}'
    options: |
      ; Workaround for: https://github.com/owncloud/core/issues/17329
      apc.enable_cli = 1
```

Actually I don't understand the reason, why `apc.enable_cli` can't be present generally.

**Comment by ypid** [comment]
Looks like you fixed this in #1856. Can you close this issue?

**Comment by jankowa** [comment]
fixed in #1856
